### DISCUSSION TITLE
copy paste error

### DISCUSSION THREAD

**Discussion by mcblacky** [GioF71/squeezelite-docker#227] [General]
### **"To Whom It May Concern" **
When playing arround with your excelent squeezelite image, I encountered a copy paste error!

I tried to run the following example copied from from the docker hub:

```
docker run \
    -it \
    --rm \
    --name squeezelite \ 
    --network host \
    -e SQUEEZELITE_NAME="home-pi" \
    -e SQUEEZELITE_AUDIO_DEVICE="hw:CARD=Headphones,DEV=0" \
    --device /dev/snd \
    giof71/squeezelite:stable
```

If you copy paste this example to your docker machine it will fail with following result:

**docker: invalid reference format.
See 'docker run --help'.
-bash: --network: command not found**

**This error is due a space character in following command option:**

```
--name squeezelite \ #<---extra space character !!!!!!!!!
--name squeezelite \#<--- when extra space character is removed the error will not be generated

```
This happens if copied from dockerhub and from github with all **docker run** commands

**Comment by GioF71** [comment]
will fix it asap, thank you

**Comment by GioF71** [comment]
This should be fixed now, also on docker hub.
Thank you for the nicely done issue report.

**Comment by mcblacky** [comment]
All correct now! And herewith I like to thank you to make this excelent docker image availlable for us!

**Comment by GioF71** [comment]
Thank you for the kind words, much appreciated.
Feel free to open discussions for new feature request also on my other repositories.
If you like some of them, consider starring if you have not done that yet.
Thank you again.
Cheers

**Reply by mcblacky** [reply]
I was unaware of starring, good you mentioned it! Just started with docker and I am impressed of the possibilties. What are the benefits of the stars? I gave you one now while I would have gave you 5 if possible!

**Reply by GioF71** [reply]
Hello, starring is a way of showing your appreciation and for recommending the repository to other users. You can mark the repositories you like by starring it. 
If you star a repo, this appears to you with the yellow star instead of the empty star. The number near the star indicates how many other users have starred the repo.
You cannot give more than one star, you just star it if you like it. Otoh of course you can star multiple repositories from the same user.
